lib/rom/sql/association/many_to_one.rb

```diff
@@ -15,6 +15,8 @@ def call(relations)
           left_schema = left.schema
           right_schema = right.schema.project_pk
 
+          join_fk = foreign_key || right.foreign_key(target.relation)
```

Comment:
ugh right, this is what happens when you don't run all specs after making changes, thanks :) fixing...